crates/core/src/factories/core_fetch.rs

```diff
@@ -368,7 +369,7 @@ impl CoreFetch {
                             .remove_peer(&peer_url);
                     }
                     Err(err) => {
-                        tracing::warn!("could not send fetch request for op {op_id} to peer {peer_url}: {err}");
+                        tracing::warn!("could not send fetch request for op {op_id} to agent {peer_url}: {err}. Putting agent on back off list.");
```

Comment:
`agent` preferred here or is this a merge thing?

Reply:
a merge thing